Dockerfile

```diff
@@ -1,31 +1,16 @@
-FROM jenkins/jenkins:2.176.2 as install_scripts_source
+FROM cloudbees/cloudbees-jenkins-distribution:2.176.2.3 as war_source
 
-FROM cloudbees/cloudbees-jenkins-distribution:2.176.2.3
-COPY --from=install_scripts_source /usr/local/bin/jenkins-support /usr/local/bin/jenkins-support
-COPY --from=install_scripts_source /usr/local/bin/install-plugins.sh /usr/local/bin/install-plugins.sh
-
-ENV JENKINS_UC http://jenkins-updates.cloudbees.com
-ENV JENKINS_INCREMENTALS_REPO_MIRROR https://repo.jenkins-ci.org/incrementals
-
-# Workaround https://github.com/jenkinsci/docker/issues/857
-USER root
-RUN mkdir -p /usr/share/jenkins/ && ln -s /usr/share/cloudbees-jenkins-distribution/cloudbees-jenkins-distribution.war /usr/share/jenkins/jenkins.war
-USER cloudbees-jenkins-distribution
+FROM jenkins/jenkins:2.176.2
+COPY --from=war_source /usr/share/cloudbees-jenkins-distribution/cloudbees-jenkins-distribution.war /usr/share/jenkins/jenkins.war
```

Comment:
Only real comment is maybe add one line of comment here to explain to the user why you are doing this. This isn't super obvious to myself who is a relatively inexperienced Jenkins/CJD user.

Reply:
Thanks for the change